datapackage.json

```diff
@@ -68,6 +77,5 @@
       "specType": "simple",
       "title": "10 year US Government Bond Yields (Monthly granuarlity)"
     }
-  ],
-  "collection": "economic-data"
```

Comment:
We need this property I think

Reply:
`datapackage.json` was changed because I ran this code https://github.com/datasets/bond-yields-us-10y/blob/main/bond_us_flow.py, and I thought those changes were necessary

Reply:
@gradedSystem Yes, but we need to keep some properties